library/core/src/cmp.rs

```diff
@@ -1234,23 +1216,7 @@ where
     F: ~const Destruct,
     K: ~const Destruct,
 {
-    cfg_if! {
-        if #[cfg(bootstrap)] {
-            const fn imp<T, F: ~const FnMut(&T) -> K, K: ~const Ord>(
-                f: &mut F,
-                (v1, v2): (&T, &T),
-            ) -> Ordering
-            where
-                T: ~const Destruct,
-                K: ~const Destruct,
-            {
-                f(v1).cmp(&f(v2))
-            }
-            min_by(v1, v2, ConstFnMutClosure::new(&mut f, imp))
-        } else {
-            min_by(v1, v2, const |v1, v2| f(v1).cmp(&f(v2)))
-        }
-    }
+    min_by(v1, v2, const |v1, v2| f(v1).cmp(&f(v2)))
```

Comment:
You could leave this bit as-is for now to keep old rustfmt, or maybe even as `cfg(rustfmt)`.

Reply:
Yes, in theory, but I think we'll have to take the cost of a nightly toolchain for now.

Reply:
I think rustc_bootstrap for rustfmt is going to be the right solution long-term, but I don't have the bandwidth to push that through right now.